ignition/chain_config.json

```diff
@@ -34,241 +34,273 @@
           "ji": 2,
           "jmp": 2,
           "jne": 2,
           "jnei": 2,
           "jnzi": 2,
-          "jmpf": 1,
-          "jmpb": 1,
-          "jnzf": 1,
-          "jnzb": 1,
-          "jnef": 1,
-          "jneb": 1,
+          "jmpf": 2,
+          "jmpb": 2,
+          "jnzf": 2,
+          "jnzb": 2,
+          "jnef": 2,
+          "jneb": 2,
           "lb": 2,
-          "log": 165,
+          "log": 80,
           "lt": 2,
           "lw": 2,
-          "mint": 29024,
+          "mint": 6566,
           "mlog": 2,
           "mod": 2,
           "modi": 2,
           "move": 2,
-          "movi": 2,
-          "mroo": 4,
+          "movi": 1,
+          "mroo": 3,
           "mul": 2,
           "muli": 2,
           "mldv": 3,
           "noop": 1,
           "not": 2,
-          "or": 2,
+          "or": 1,
           "ori": 2,
-          "poph": 3,
-          "popl": 3,
-          "pshh": 4,
-          "pshl": 4,
-          "ret_contract": 134,
-          "rvrt_contract": 153,
+          "poph": 2,
+          "popl": 2,
+          "pshh": 3073,
+          "pshl": 3016,
+          "ret_contract": 43,
+          "rvrt_contract": 39,
           "sb": 2,
           "sll": 2,
           "slli": 2,
           "srl": 2,
           "srli": 2,
-          "srw": 209,
+          "srw": 237,
           "sub": 2,
           "subi": 2,
           "sw": 2,
-          "sww": 22501,
-          "time": 50,
-          "tr": 33912,
-          "tro": 24294,
+          "sww": 5708,
+          "time": 106,
+          "tr": 9253,
+          "tro": 7199,
           "wdcm": 2,
-          "wqcm": 3,
-          "wdop": 3,
+          "wqcm": 2,
+          "wdop": 2,
           "wqop": 3,
           "wdml": 3,
-          "wqml": 4,
-          "wddv": 5,
-          "wqdv": 6,
-          "wdmd": 10,
-          "wqmd": 17,
-          "wdam": 9,
-          "wqam": 11,
-          "wdmm": 10,
-          "wqmm": 10,
+          "wqml": 3,
+          "wddv": 3,
+          "wqdv": 4,
+          "wdmd": 6,
+          "wqmd": 9,
+          "wdam": 6,
+          "wqam": 6,
+          "wdmm": 6,
+          "wqmm": 6,
           "xor": 2,
           "xori": 2,
+          "aloc": {
+            "LightOperation": {
+              "base": 2,
+              "units_per_gas": 35
+            }
+          },
+          "bsiz": {
+            "LightOperation": {
+              "base": 25,
+              "units_per_gas": 564
+            }
+          },
+          "bldd": {
+            "LightOperation": {
+              "base": 33,
+              "units_per_gas": 130
+            }
+          },
+          "cfe": {
+            "LightOperation": {
+              "base": 10,
+              "units_per_gas": 62
+            }
+          },
+          "cfei": {
+            "LightOperation": {
+              "base": 10,
+              "units_per_gas": 66
+            }
+          },
           "call": {
             "LightOperation": {
-              "base": 18190,
```

Comment:
What caused the substantial drop in price? Were the original prices based on the old VPS vs the new bare metal reference machine?

Reply:
It is because of optimization for SMT that we added in the `fuel-vm 0.56` to not fetch sibling nodes. It improved performance for all opcodes that modified the storage in two times